src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -303,4 +305,33 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
         return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
     }
+
+    private mountComplianceSection(sellingEntity: string) {
+        const complianceDiv = document.getElementById('compliance');
+
+        const complianceOptions: DigitalRiverElementOptions = {
+            classes: {
+                base: 'DRElement',
+            },
+            compliance: {
+                entity: sellingEntity,
+            },
+        };
+
+        if (complianceDiv) {
+            complianceDiv.innerHTML = '';
+
+            const complianceElement = this._getDigitalRiverJs().createElement('compliance', complianceOptions);
+            complianceElement.mount('compliance');
+        } else {
+            const drfooter = document.createElement('div');
+            drfooter.setAttribute('id', 'compliance');
+            drfooter.style.cssText = 'min-height: 45px;';
+            drfooter.classList.add('layout');
```

Comment:
Could we provide more context for the style here? Why `min-height: 45px`, `layout`?

Reply:
Hi @bc-peng Actually this code was sent by the provider, due to their requirements.

Reply:
Understood. Thanks. 🙏🏻